submitted_models/cerberus_anymal_c_sensor_config_1/specifications.md

```diff
@@ -0,0 +1,90 @@
+<!--- This is a Markdown description of a robot model submitted for inclusion in the
+DARPA Subterranean Challenge Technology Repository -->
+# ANYmal
+This specifications.md file is a description and proof of virtual model validation for the CERBERUS ANYmal C with Sensor Configuration 1. This robot may be launched using a `ign launch` command with the variable name `cerberus_anymal_c_sensor_config_1`.
+
+## Description
+ANYmal is a highly sophisticated four-legged robot, designed to tackle the challenges of harsh search and rescue operations, inspections, and other surveillance duties.
+In this configuration ANYmal is equipped with the following sensors: LIDAR, two downward facing half-sphere LIDARs, IMU and a perception unit including seven cameras and an additional IMU.
+
+## Dependencies
+To be able to control this robot model you need a few additional packages. Please clone the repository [`cerberus_anymal_locomotion`](https://github.com/leggedrobotics/cerberus_anymal_locomotion) and follow the instructions specified in its README.
+
+## Usage Instructions
+The vehicle can be launched with the following command:
+```
+ign launch -v 4 competition.ign robotName1:=anymal_c robotConfig1:=CERBERUS_ANYMAL_C_SENSOR_CONFIG_1
+```
+
+In another terminal you can source the "anymal_locomotion_ws" workspace (see [Dependencies](#markdown-header-dependencies)) to start the locomotion controller node:
+```
+cd ~/anymal_locomotion_ws/
+source devel/setup.bash
+roslaunch cerberus_anymal_c_control_1 cerberus_anymal_controller.launch
+```
+
+At this point you can control the vehicle model with a twist command, for example:
+```
+rostopic pub /anymal_c/cmd_vel geometry_msgs/Twist "linear:
+  x: 0.5
+  y: 0.0
+  z: 0.0
+angular:
+  x: 0.0
+  y: 0.0
+  z: 0.0"
+```
+
```

Comment:
Please add a note mentioning that CERBERUS commands _negative X velocity to move forward_ on the ANYmal due to the positioning of the mounted sensors.